apps/organization/interfaces/organization.interface.ts

```diff
@@ -119,13 +119,6 @@ interface IOrganizationPagination {
   logoUrl: string;
 }
 
-export interface IOrganizationDashboard {
-  usersCount: number,
-  schemasCount: number,
```

Comment:
why did you remove this. Could you please tell me?

Reply:
We are using this interface in api-gateway and organisation service, so I moved it to a common file.